--- origsrc/gtk+extra-2.1.2/configure.in	2011-01-31 02:40:55.168491900 -0600
+++ src/gtk+extra-2.1.2/configure.in	2011-01-31 02:42:27.380925200 -0600
@@ -94,7 +94,7 @@ if test "$os_win32" = "yes"; then
 fi
 AM_CONDITIONAL(MS_LIB_AVAILABLE, test x$ms_librarian = xyes)
 
-if test "$platform_win32" = "yes"; then
+if test "$os_win32" = "yes"; then
   gdktarget=win32
 else
   gdktarget=x11
@@ -155,7 +155,7 @@ dnl header file checks
 AC_CHECK_HEADERS(unistd.h, AC_DEFINE(HAVE_UNISTD_H))
 AC_CHECK_HEADERS(dirent.h, AC_DEFINE(HAVE_DIRENT_H))
 dnl Don't attempt to use fnmatch under win32 due to problems with creating DLLs.
-if test "$platform_win32" = "no"; then
+if test "$os_win32" = "no"; then
   AC_CHECK_HEADERS(fnmatch.h, AC_DEFINE(HAVE_FNMATCH_H))
 fi
 
