diff -urN -x .build -x .inst -x .sinst -x configure -x Makefile.in -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x 'intltool*' -x autom4te.cache gtk+extra-2.1.1-orig/CYGWIN-PATCHES/README gtk+extra-2.1.1/CYGWIN-PATCHES/README
--- gtk+extra-2.1.1-orig/CYGWIN-PATCHES/README	1969-12-31 18:00:00.000000000 -0600
+++ gtk+extra-2.1.1/CYGWIN-PATCHES/README	2005-08-01 20:32:46.734375000 -0500
@@ -0,0 +1,178 @@
+gtkextra2-x11
+-------------------------------------------
+GTK+extra is a useful widget set complementary to GTK+ for creating
+graphical interfaces for the X Windows System. It is written in C and
+requires GTK+ version 2.x.
+
+Runtime requirements:
+  cygwin-1.5.18-1
+  atk-runtime-1.8.0-1
+  expat-1.95.8-1
+  glib2-runtime-2.4.8-1
+  gtk2-x11-runtime-2.4.14-1
+  libXft2-2.1.6-1
+  libfontconfig1-2.2.2-1
+  libfreetype26-2.1.5-1
+  libiconv2-1.9.2-1
+  libintl3-0.14.1-1
+  pango-runtime-1.6.0-2
+  xorg-x11-bin-dlls-6.8.2.0-1
+  zlib-1.2.2-2
+
+Build requirements:
+  cygwin-1.5.18-1
+  autoconf2.5-2.59-1
+  automake1.7-1.7.9-2
+  binutils-20050610-1
+  gawk-3.1.5-1
+  gcc-core-3.4.4-1
+  libtool1.5-1.5.18-1
+  make-3.80-1
+  sed-4.1.4-1
+
+  atk-runtime-1.8.0-1
+  expat-1.95.8-1
+  gettext-0.14.1-1
+  glib2-devel-2.4.8-1
+  gtk2-x11-devel-2.4.14-1
+  libXft-devel-2.1.6-1
+  libfontconfig-devel-2.2.2-1
+  libfreetype2-devel-2.1.5-1
+  libiconv-1.9.2-1
+  pango-devel-1.6.0-2
+  xorg-x11-devel-6.8.2.0-1
+  zlib-1.2.2-2
+
+Canonical website:
+  http://gtkextra.sourceforge.net/
+
+Canonical download:
+  http://prdownloads.sourceforge.net/scigraphica/
+
+-------------------------------------------
+
+Build instructions:
+  unpack gtkextra2-x11-2.1.1-1-src.tar.bz2
+    if you use setup to install this src package, it will be
+	 unpacked under /usr/src automatically
+  cd /usr/src
+  ./gtk+extra-2.1.1-1.sh all
+
+This will create:
+  /usr/src/gtkextra2-x11-2.1.1-1-src.tar.bz2
+  /usr/src/gtkextra2-x11-2.1.1-1.tar.bz2
+
+-------------------------------------------
+
+Files included in the binary package:
+
+  /usr/bin/cyggtkextra-x11-2.0-1.dll
+  /usr/include/gtkextra-2.0/gtkextra/gtkbordercombo.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkcharsel.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkcheckitem.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkcolorcombo.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkcombobutton.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkdirtree.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkextra-marshal.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkextra.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkextrafeatures.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkextraicons.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkextratypebuiltins.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkfileicons.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkfilelist.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkfontcombo.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkiconfilesel.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkiconlist.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkitementry.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkplot.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkplot3d.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkplotarray.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkplotbar.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkplotbox.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkplotbubble.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkplotcandle.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkplotcanvas.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkplotcanvasellipse.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkplotcanvasline.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkplotcanvaspixmap.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkplotcanvasplot.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkplotcanvasrectangle.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkplotcanvastext.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkplotcsurface.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkplotdata.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkplotdt.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkplotflux.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkplotgdk.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkplotpc.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkplotpixmap.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkplotpolar.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkplotprint.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkplotps.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkplotsegment.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkplotsurface.h
+  /usr/include/gtkextra-2.0/gtkextra/gtkpsfont.h
+  /usr/include/gtkextra-2.0/gtkextra/gtksheet.h
+  /usr/include/gtkextra-2.0/gtkextra/gtktogglecombo.h
+  /usr/lib/libgtkextra-x11-2.0.dll.a
+  /usr/lib/libgtkextra-x11-2.0.la
+  /usr/lib/pkgconfig/gtkextra-2.0.pc
+  /usr/share/doc/Cygwin/gtkextra2-x11-2.1.1.README
+  /usr/share/doc/gtkextra2-x11-2.1.1/AUTHORS
+  /usr/share/doc/gtkextra2-x11-2.1.1/ChangeLog
+  /usr/share/doc/gtkextra2-x11-2.1.1/COPYING
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/gtkextra-bordercombo.html
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/gtkextra-charsel.html
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/gtkextra-checkitem.html
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/gtkextra-colorcombo.html
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/gtkextra-combobox.html
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/gtkextra-csurface.html
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/gtkextra-dirtree.html
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/gtkextra-filelist.html
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/gtkextra-fontcombo.html
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/gtkextra-general.html
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/gtkextra-iconfileselection.html
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/gtkextra-iconlist.html
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/gtkextra-itementry.html
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/gtkextra-plot.html
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/gtkextra-plot3d.html
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/gtkextra-plotbar.html
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/gtkextra-plotbox.html
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/gtkextra-plotcanvas.html
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/gtkextra-plotdata.html
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/gtkextra-plotdt.html
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/gtkextra-plotflux.html
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/gtkextra-plotpc.html
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/gtkextra-plotpixmap.html
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/gtkextra-plotpolar.html
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/gtkextra-plotprint.html
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/gtkextra-plotps.html
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/gtkextra-plotstructures.html
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/gtkextra-plotsurface.html
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/gtkextra-psfont.html
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/gtkextra-sheet.html
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/gtkextra-sheetstructures.html
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/gtkextra-togglecombo.html
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/gtkextra-version.html
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/index.html
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/pics/gtk_border_combo.jpg
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/pics/gtk_charsel.png
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/pics/gtk_color_combo.jpg
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/pics/gtk_conlist_workbench.jpg
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/pics/gtk_filelist_filter.jpg
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/pics/gtk_filelist_new.jpg
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/pics/gtk_filelist_tree.jpg
+  /usr/share/doc/gtkextra2-x11-2.1.1/html/pics/gtk_iconlist_plot.jpg
+  /usr/share/doc/gtkextra2-x11-2.1.1/INSTALL
+  /usr/share/doc/gtkextra2-x11-2.1.1/NEWS
+  /usr/share/doc/gtkextra2-x11-2.1.1/README
+  /usr/share/doc/gtkextra2-x11-2.1.1/TODO
+
+------------------
+
+Port Notes:
+
+----- version 2.1.1-1 -----
+Initial release for Cygwin.
+
+
+Cygwin port maintained by:  http://cygwin-ports.sourceforge.net/
diff -urN -x .build -x .inst -x .sinst -x configure -x Makefile.in -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x 'intltool*' -x autom4te.cache gtk+extra-2.1.1-orig/CYGWIN-PATCHES/setup.hint gtk+extra-2.1.1/CYGWIN-PATCHES/setup.hint
--- gtk+extra-2.1.1-orig/CYGWIN-PATCHES/setup.hint	1969-12-31 18:00:00.000000000 -0600
+++ gtk+extra-2.1.1/CYGWIN-PATCHES/setup.hint	2005-08-01 20:32:47.140625000 -0500
@@ -0,0 +1,6 @@
+category: Gnome X11
+requires: cygwin gtk2-x11-runtime
+sdesc: "Additional graphing widgets for GTK+ 2.x"
+ldesc: "GTK+extra is a useful widget set complementary to GTK+ for creating
+graphical interfaces for the X Windows System. It is written in C and
+requires GTK+ version 2.x."
diff -urN -x .build -x .inst -x .sinst -x configure -x Makefile.in -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x 'intltool*' -x autom4te.cache gtk+extra-2.1.1-orig/configure.in gtk+extra-2.1.1/configure.in
--- gtk+extra-2.1.1-orig/configure.in	2005-06-24 12:34:06.000000000 -0500
+++ gtk+extra-2.1.1/configure.in	2005-08-01 18:46:51.093750000 -0500
@@ -93,7 +93,7 @@
 fi
 AM_CONDITIONAL(MS_LIB_AVAILABLE, test x$ms_librarian = xyes)
 
-if test "$platform_win32" = "yes"; then
+if test "$os_win32" = "yes"; then
   case " $CFLAGS " in
   *[\ \	]-fnative-struct[\ \	]*) ;;
   *) CFLAGS="$CFLAGS -fnative-struct" ;;
@@ -156,7 +156,7 @@
 AC_CHECK_HEADERS(unistd.h, AC_DEFINE(HAVE_UNISTD_H))
 AC_CHECK_HEADERS(dirent.h, AC_DEFINE(HAVE_DIRENT_H))
 dnl Don't attempt to use fnmatch under win32 due to problems with creating DLLs.
-if test "$platform_win32" = "no"; then
+if test "$os_win32" = "no"; then
   AC_CHECK_HEADERS(fnmatch.h, AC_DEFINE(HAVE_FNMATCH_H))
 fi
 
diff -urN -x .build -x .inst -x .sinst -x configure -x Makefile.in -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x 'intltool*' -x autom4te.cache gtk+extra-2.1.1-orig/gtkextra/gtkextratypebuiltins.c gtk+extra-2.1.1/gtkextra/gtkextratypebuiltins.c
--- gtk+extra-2.1.1-orig/gtkextra/gtkextratypebuiltins.c	2005-06-24 12:52:27.000000000 -0500
+++ gtk+extra-2.1.1/gtkextra/gtkextratypebuiltins.c	2005-08-01 19:06:10.515625000 -0500
@@ -21,6 +21,7 @@
   return etype;
 }
 
+
 /* enumerations from "gtkplot.h" */
 GType
 gtk_plot_scale_get_type (void)
@@ -36,6 +37,7 @@
   }
   return etype;
 }
+
 GType
 gtk_plot_symbol_type_get_type (void)
 {
@@ -61,6 +63,7 @@
   }
   return etype;
 }
+
 GType
 gtk_plot_symbol_style_get_type (void)
 {
@@ -76,6 +79,7 @@
   }
   return etype;
 }
+
 GType
 gtk_plot_border_style_get_type (void)
 {
@@ -91,6 +95,7 @@
   }
   return etype;
 }
+
 GType
 gtk_plot_line_style_get_type (void)
 {
@@ -110,6 +115,7 @@
   }
   return etype;
 }
+
 GType
 gtk_plot_connector_get_type (void)
 {
@@ -128,6 +134,7 @@
   }
   return etype;
 }
+
 GType
 gtk_plot_label_pos_get_type (void)
 {
@@ -143,6 +150,7 @@
   }
   return etype;
 }
+
 GType
 gtk_plot_error_get_type (void)
 {
@@ -157,6 +165,7 @@
   }
   return etype;
 }
+
 GType
 gtk_plot_orientation_get_type (void)
 {
@@ -172,6 +181,7 @@
   }
   return etype;
 }
+
 GType
 gtk_plot_axis_pos_get_type (void)
 {
@@ -188,6 +198,7 @@
   }
   return etype;
 }
+
 GType
 gtk_plot_label_style_get_type (void)
 {
@@ -203,6 +214,7 @@
   }
   return etype;
 }
+
 GType
 gtk_plot_ticks_pos_get_type (void)
 {
@@ -219,6 +231,7 @@
   return etype;
 }
 
+
 /* enumerations from "gtkplot3d.h" */
 GType
 gtk_plot_plane_get_type (void)
@@ -238,6 +251,7 @@
   }
   return etype;
 }
+
 GType
 gtk_plot_side_get_type (void)
 {
@@ -257,6 +271,7 @@
   return etype;
 }
 
+
 /* enumerations from "gtkplotbar.h" */
 GType
 gtk_plot_bar_units_get_type (void)
@@ -274,6 +289,7 @@
   return etype;
 }
 
+
 /* enumerations from "gtkplotcsurface.h" */
 GType
 gtk_plot_projection_get_type (void)
@@ -291,6 +307,7 @@
   return etype;
 }
 
+
 /* enumerations from "gtkplotcanvas.h" */
 GType
 gtk_plot_canvas_action_get_type (void)
@@ -308,6 +325,7 @@
   }
   return etype;
 }
+
 GType
 gtk_plot_canvas_flag_get_type (void)
 {
@@ -323,6 +341,7 @@
   }
   return etype;
 }
+
 GType
 gtk_plot_canvas_selection_get_type (void)
 {
@@ -338,6 +357,7 @@
   }
   return etype;
 }
+
 GType
 gtk_plot_canvas_selection_mode_get_type (void)
 {
@@ -352,6 +372,7 @@
   }
   return etype;
 }
+
 GType
 gtk_plot_canvas_pos_get_type (void)
 {
@@ -375,6 +396,7 @@
   return etype;
 }
 
+
 /* enumerations from "gtkplotpc.h" */
 GType
 gtk_plot_page_size_get_type (void)
@@ -393,6 +415,7 @@
   }
   return etype;
 }
+
 GType
 gtk_plot_page_orientation_get_type (void)
 {
@@ -407,6 +430,7 @@
   }
   return etype;
 }
+
 GType
 gtk_plot_units_get_type (void)
 {
@@ -424,6 +448,7 @@
   return etype;
 }
 
+
 /* enumerations from "gtkplotsegment.h" */
 GType
 gtk_plot_arrow_get_type (void)
@@ -441,6 +466,7 @@
   return etype;
 }
 
+
 /* enumerations from "gtksheet.h" */
 GType
 gtk_sheet_attr_type_get_type (void)
@@ -463,6 +489,7 @@
   return etype;
 }
 
+
 /* enumerations from "gtkplotcanvasline.h" */
 GType
 gtk_plot_canvas_arrow_get_type (void)
@@ -480,6 +507,7 @@
   return etype;
 }
 
+
 /* enumerations from "gtkplotcanvasplot.h" */
 GType
 gtk_plot_canvas_plot_pos_get_type (void)
@@ -502,5 +530,6 @@
   return etype;
 }
 
+
 /* Generated data ends here */
 
